site/tests/moderation/test_moderation_queries.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test ModerationQuery model class."""
+
+
+from invenio_db import db
+from invenio_search import current_search_client
+from zenodo_rdm.moderation.models import ModerationQuery
+
+
+def test_moderation_query_creation(app):
```

Comment:
minor/shelve: I trust the feature works, but if relatively easy I would also add a test that performs a match query.